src/index.js

```diff
@@ -21,11 +42,10 @@ const ReactNative = {
   Touchable,
   View,
 
-  // apis
-  StyleSheet,
-
   // React
-  ...React
+  ...React,
```

Comment:
could spread ReactDOM too, effectively returning to the 0.13 API since we're web-only